apps/ledger-live-desktop/src/renderer/components/NewCarousel/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Icons } from "@ledgerhq/react-ui";
+import useEmblaCarousel, { UseEmblaCarouselType } from "embla-carousel-react";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import Footer from "./footer";
+
+type Props = {
+  slides: {
+    id: number | string;
+    Component: () => React.JSX.Element;
+  }[];
+};
+
+const CarouselContainer = styled.div`
+  padding: 12px 16px 12px 16px;
+`;
+
+const Embla = styled.div`
+  overflow: hidden;
+`;
+
+const EmblaContainer = styled.div`
+  display: flex;
+`;
+
+const EmblaSlide = styled.div`
+  flex: 0 0 100%;
+  min-width: 0;
+`;
+
+const Carousel = ({ slides }: Props) => {
```

Comment:
This component might belong to our Design System.
Why do we introduce a new component to do the same as https://react-ui-storybook.vercel.app/?path=/docs/cards-portfolio-carousel--docs?
If the design evolves we should update the previous component.

Reply:
Agreed. I was thinking of doing that in another PR if that's alright.

Reply:
I added it to the DS.